app/views/admin/users/_member_index.html.erb

```diff
@@ -7,34 +7,62 @@
 
     <!-- Small screen data view start -->
     <div class="block m:hidden fs-s">
-      <h2 class="member-data-heading fs-s py-2 -mx-4 px-6 color-base-60">Members</h2>
-      <ul class="list-none mx-2">
-        <% @users.each do |user| %>
+      <h2 class="member-data-heading fs-s py-2 -mx-3 px-3 color-base-60">Members</h2>
+      <ul class="list-none">
+        <% @users.includes([:organization_memberships]).each do |user| %>
           <li class="py-4">
             <article>
               <header class="flex">
-                <img class="radius-full mr-2" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="" />
-                <div class="flex flex-col">
-                  <h3 class="fs-base"><%= link_to user.name, admin_user_path(user) %></h3>
-                  <span class="color-base-60">@<%= user.username %></span>
+                <a href="<%= admin_user_path(user.id) %>">
+                  <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
+                </a>
+                <div class="pl-3 flex-1 fs-s"><h3 class="fs-base"><%= link_to user.name, admin_user_path(user) %></h3>
+                  <p class="color-base-60"><%= link_to "@#{user.username}", admin_user_path(user) %>&nbsp;
+                  </p>
                 </div>
               </header>
               <div class="flex justify-between py-2">
-                <div>standing info</div>
-                <div>role info</div>
+                <div>
+                <% if user.suspended? %>
+                  <span class="c-indicator mr-2 inline-block" style="--bg: #DC2626"></span>Suspended
+                <% elsif user.warned? %>
+                  <span class="c-indicator mr-2 inline-block" style="--bg: #F59E0B"></span>Warned
+                <% elsif user.comment_suspended? %>
+                  <span class="c-indicator mr-2 inline-block" style="--bg: #DC2626"></span>Comment suspended
+                <% elsif user.trusted? %>
+                  <span class="c-indicator mr-2 inline-block" style="--bg: #059669"></span>Trusted
+                <% else %>
+                  <span class="c-indicator mr-2 inline-block" style="--bg: #A3A3A3"></span>Good standing
+                <% end %>
```

Comment:
(Suggestion): Perhaps we can move this out into a partial? and reference the partial for line 102 to 112 as well.

Reply:
@Ridhwana I tried doing this, but run into nested loop troubles, so I pulled moving this out into a partial as a cleanup task within the overarching issue for this PR so that it wouldn't hold the work up. 😕